docs/topics/azure-stack.md

```diff
@@ -15,25 +15,23 @@
 * [Known Issues and Limitations](#known-issues-and-limitations)
 * [Frequently Asked Questions](#frequently-asked-questions)
 
-## Introduction
+## Introduction to the AKS Engine for Azure Stack
 
 Starting from [release](https://github.com/Azure/aks-engine/releases/) v0.36.2, AKS Engine can be used to provision self-managed Kubernetes clusters on Azure Stack. It is now possible to execute AKS Engine's `generate`, `deploy`, `upgrade`, and `scale` commands as if you were targeting Azure's public cloud. You are only required to slightly update your cluster definition to provide some extra information about your Azure Stack instance.
 
 The goal of this guide is to explain how to provision Kubernetes clusters to Azure Stack using AKS Engine and to capture the differences between Azure and Azure Stack. Bear in mind as well that not every AKS Engine feature or configuration option is currently supported on Azure Stack. In most cases, these are not available because dependent Azure components are not part of Azure Stack.
 
-## Service Principals and Identity Providers
+## Service principals and identity providers
```

Comment:
Is there a style guide that recommends capitalizing only the first word of a header? I've always used newspaper-style title case, but I am old.

Your choice seems to be common lately, so I'm curious. There are a mix of these styles in our several upstream Kubernetes projects.

Reply:
@mboersma The Azure / Microsoft style guide uses sentence case for headings. If I had my say, I would use title case for headings and titles...

Reply:
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize is clear:

> Default to sentence-style capitalization—capitalize only the first word of a heading or phrase and any proper nouns or names. Never Use Title Capitalization (Like This). Never Ever.

I'm happy to comply. Apparently I hadn't consulted our own style guide before. 🙈